src/sass/pages/index.scss

```diff
@@ -0,0 +1,44 @@
+.home-hero {
+  @include corporate-gradient;
+  color: $color-hero-text;
+  padding: $padding-hero-vertical 0;
+
+  &__wrap {
+    @include row;
+    @include container;
+  }
+
+  &__info {
+    @include columns(5);
+
+    h1 {
+      font-size: $font-size-title-big;
```

Comment:
Isn't there any global style for h1 and such?
It would be useful to have something like
`@include title-big` that would set the size, line-height, capitalization and such.
Otherwise different titles could be out of sync